components/script/dom/windowproxy.rs

```diff
@@ -148,7 +149,7 @@ impl WindowProxy {
                 ((*get_object_class(window_jsobject.get())).flags & JSCLASS_IS_GLOBAL),
                 0
             );
-            let _ac = JSAutoRealm::new(cx, window_jsobject.get());
+            let _ac = enter_realm(window);
```

Comment:
Let's revert the changes in this file. We already have the cx and JSObject values, so calling this method to reacquire them is redundant.